blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_7.py

```diff
@@ -0,0 +1,44 @@
+"""Formula 8.7 from NEN-EN 1992-1-1+C2:2011: Chapter 8: Detailing of reinforcement and prestressing tendons"""
+# pylint: disable=arguments-differ
+# pylint: disable=duplicate-code
+
+from blueprints.codes.eurocode.nen_en_1992_1_1_c2_2011 import NEN_EN_1992_1_1_C2_2011
+from blueprints.codes.formula import Formula
+from blueprints.type_alias import MM
+from blueprints.validations import raise_if_negative
+
+
+class Form8Dot7MinimumCompressionAnchorage(Formula):
+    """Class representing formula 8.7 for the calculation of the minimum anchorage length if no other limitation is applied for anchorage in
+    compression."""
+
+    label = "8.7"
+    source_document = NEN_EN_1992_1_1_C2_2011
+
+    def __init__(
+        self,
+        l_b_rqd: MM,
+        phi: MM,
```

Comment:
I don't believe this is a phi, I think it is a diameter symbol. Why wouldn't we just call it as what it is: `diameter`

Same for l_b_rqd --> isn't it more readable to call it: basic_achorage_length_required?

@egarciamendez , @johan-tuls : What is your take on this?

Reply:
The reason we're maintaining the same notation in our formulas as in the Eurocode is due to a consensus reached by the entire group in our last meeting. Also, keep in mind that each formula is being thoroughly documented.

Reply:
it helps if variable names are self documenting IMO. in this case I dont believe the EC intents it to be a phi: look into the symbol nomenclature: there are 3 variants of the symbol phi -> are we adhering to that too? 🤣 (not saying we have to) in this case I think the idea is an O with dash symbol: Ø

Reply:
I think in this case it should be "diamater" instead of phi. It will be misleading otherwise.